src/utils/V3Utils.ts

```diff
@@ -12,13 +12,23 @@ import {
   V3SpeedUp,
 } from "../interfaces";
 import { BN } from "./BigNumberUtils";
-import { isDefined } from "./TypeGuards";
 
-type Deposit = V2Deposit | V3Deposit;
 type Fill = V2Fill | V3Fill;
-type SpeedUp = V2SpeedUp | V3SpeedUp;
-type RelayData = V2RelayData | V3RelayData;
-type SlowFillLeaf = V2SlowFillLeaf | V3SlowFillLeaf;
```

Comment:
ooc, is `Fill` intentionally the only locally-defined union that wasn't removed?